### PR TITLE
HNS configurable from command line

### PR DESCRIPTION
**Pull request by m-kostrzewa**
Lets specify hns network, gateway, and which net adapter to use as command line arguments.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 	"github.com/docker/go-plugins-helpers/network"
 	"github.com/docker/go-plugins-helpers/sdk"
 )
 
 func main() {
-	d, err := driver.NewDriver()
-	if err != nil {
+	var subnet = flag.String("subnet", "172.117.0.0/16", "subnet in CIDR format for HNS")
+	var gateway = flag.String("gateway", "172.117.0.1", "default gateway IP for HNS")
+	var adapter = flag.String("adapter", "Ethernet0",
```
Comment (Attil) [inline main.go:15]: What is `Ethernet0`? `Get-NetAdapter | format-custom` doesn't contain such a string, how do i get it for my desired adapter?

**Review by Attil — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (m-kostrzewa) [reply main.go:15]: It depends, Get-NetAdapter returns Ethernet0 on my vm

Comment (Attil) [reply main.go:15]: In that case, i think a good idea would be to pick one of existing net adapters? There's no `Ethernet0` on my VM, so I think this driver wouldn't work there. I have default interfaces names there.

Comment (m-kostrzewa) [reply main.go:15]: Created a minor task for this feature JW-53

**Comment by Attil** [comment]
Resolve the conflicts and you can merge.

**Pull request merged**